addons/monit/oom_immunize_process_names/process_names_to_immunize

```diff
@@ -0,0 +1,10 @@
+#Put one unique command line argument for "pgrep -f" to UNIQUELY find. Comments and empty lines are skipped.
+
+#MySQL
+/usr/libexec/mysqld
+
+#Accounting
+/usr/local/pf/var/conf/httpd.conf.d/httpd.aaa
```

Comment:
I think its because thats the only way to get the httpd.aaa process via pgrep.

@tlaurion can confirm..